workflow-templates/im-build-db-ci.yml

```
@@ -0,0 +1,186 @@
+        with:
+          db-server-name: ${{ env.DB_SERVER_NAME }}
+          db-name: ${{ env.DB_NAME }}
+          install-mock-db-objects: true # TODO: Verify this is needed
+          mock-db-object-nuget-feed-url: https://artifacts.mktp.io/artifactory/api/nuget/nuget-all # TODO: Verify this is needed and correct
+          drop-db-after-build: false
+          run-tests: true # TODO: Remove this if your database doesn't have tests.
+
+      #########################################################################################################################
+      # The previous three steps are all that most builds will need.
+      # They will build your database on the Action Runner and run any tests you have.
+      # The steps that come next aren't going to be needed by every repo, so feel free to remove what is unnecessary for yours.
+      #########################################################################################################################
+
+
+      # The next four steps can be removed if your build doesn't need to create snapshots of database objects.
+      # Otherwise they can be uncommented and the TODO's addressed.
+
+      # TODO: Replace what is in this step with what your build needs in order to determine what database objects have changed
+      # - name: Get db objects that have changed
+      #   id: changed-objects
+      #   shell: pwsh
+      #   run: |
+      #     $changedObjectsQuery = 
+      #       "SELECT
+      #           objectId,
+      #           schemaName,
+      #           objectName,
+      #           CASE objectType
+      #             WHEN 'U' THEN 'Tables'
+      #             WHEN 'V' THEN 'Views'
+      #             WHEN 'P' THEN 'StoredProcedures'
+      #             WHEN 'SO' THEN 'Sequences'
+      #             WHEN 'TF' THEN 'UserDefinedFunctions'
+      #             WHEN 'FN' THEN 'UserDefinedFunctions'
+      #             WHEN 'IF' THEN 'UserDefinedFunctions'
+      #             WHEN 'SN' THEN 'Synonyms'
+      #             ELSE objectType
+      #           END AS objectType,
+      #           operationType
+      #         FROM DBA.V_ChangedObjectsForSnapshot"
+              
+      #     $changedObjects = Invoke-Sqlcmd -ServerInstance $instanceName -Database $dbName -Query $changedObjectsQuery
+      #     $mappedObjects = $changedObjects | foreach-object { @{ schemaName=$_.schemaName; objectName=$_.objectName; objectType=$_.objectType; operationType=$_.operationType } }
+      #     $objectsAsJson = $mappedObjects | ConvertTo-Json -Compress
+
+      #     echo "::set-output name=json::$objectsAsJson"
+
+      # - name: Increment snapshots
+      #   uses: im-open/increment-database-object-snapshots@v1.0.0
+      #   with:
+      #     db-name: ${{ env.DB_NAME }}
+      #     instance-name: ${{ env.DB_SERVER_NAME }}
+      #     snapshot-path: ${{ env.SNAPSHOT_PATH }}
+      #     objects-to-increment: "${{ steps.changed-objects.outputs.json }}"
```

I think our standard is to do single quotes instead of double.  There are a number of places in this workflow file that using " instead of ' that you may want to fix.  @hpractv and I are both using YAML by Red Hat if that helps.    It may just be showing up like that though because they are in comments.
```suggestion
      #     objects-to-increment: '${{ steps.changed-objects.outputs.json }}'
```